content/automate/ManageIQ/System/CommonMethods/QuotaMethods.class/__methods__/requested.rb

```diff
@@ -146,6 +146,11 @@ def vmdb_object(model, id)
   $evm.vmdb(model, id.to_i) if model && id
 end
 
+def get_disk_size(disk_name)
+  mydisk = $evm.vmdb('disk').find_by(:filename => disk_name)
```

Comment:
@billfitzgerald0120 
Should we log disk_names that are not found and raise an error?
Also add a test case where the disk_name doesn't exist.

Reply:
@mkanoor Made changes as requested.